src/Blocks/BlockPressurePlate.h

```diff
@@ -28,7 +28,8 @@ class cBlockPressurePlateHandler :
 			return false;
 		}
 
-		return (cBlockInfo::FullyOccupiesVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)));
+		return (cBlockInfo::FullyOccupiesVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ))  ||
+				cBlockInfo::FullyOccupiesTopVoxel(a_Chunk.GetBlock(a_RelX, a_RelY - 1, a_RelZ)) );
```

Comment:
Alright. Are there any other blocks that you know of that should be able to have a pressure plate on top of them that aren't able to? I looked around, but couldn't find any. If they don't exist, it's not really any extra work to create `BlockFence.h`.